test/test_multiple_decorators.py

```diff
@@ -0,0 +1,36 @@
+from cyberbrain import Binding, Return, Symbol
+
+
+def test_multiple_decorators(trace):
+    def my_decorator(f):
+        def inner(*args):
+            a = 1
+            f(*args)
+            b = a
+
+        return inner
+
+    @my_decorator
+    @my_decorator
+    @my_decorator
+    @trace
+    def original_func():
```

Comment:
We want to test the lineno of parameters are correct for decorated functions, while this function doesn't have parameters. See other tests you modified for examples.